### ISSUE TITLE
I2C BUS 1 FAIL

### ISSUE THREAD

**Issue by gorkaarce** [alanbjohnston/CubeSatSim#284]
Hi there, 
my students and I are doing this CubeSatSim project, but we´ve been facing an issue in the last days regarding the INA sensors through BUS 1.
Both using telemetry via Foxtelem or reading directly through terminal we get 0V values for +X, +Y, PSU and Battery.
On the other hand BUS 3 is getting ok values, tipycally reading 0.9V with no panels attached and 3,5-4,5 Volts with panels attached. (see pic. attached)
We have activated manually BUS 1 through terminal following these instructions (https://www.instructables.com/Raspberry-PI-Multiple-I2c-Devices/).
We have checked I2C adresses,removed and tried with 4k7 resistors, checked jumpers, 
and now we get OK status on I2C BUS 1 in fox telem (without the resistors, but sadly no readings show up..
We are building 2 Cubesats and get same trouble on both !

Another issue with the INA testing is that we don´t get readings performing the presoldering test (see pic 2), could these 2 be related?

Any help regarding this issue would be much appreciated!
Thank you so much
Gorka

![IMAGE 2023-10-25 07:53:33](https://github.com/alanbjohnston/CubeSatSim/assets/148939068/d9e40898-74d4-4ca6-8ac7-446835a63134)

![Captura de pantalla 2023-10-25 a las 8 01 18](https://github.com/alanbjohnston/CubeSatSim/assets/148939068/80baf704-1c0e-415a-9353-83d2ef64f83d)

**Comment by alanbjohnston** [comment]
Hi Gorka,

I’d suggest you post problems like this on the Discussions page rather than as a code issue https://github.com/alanbjohnston/CubeSatSim/discussions 

It sounds like your INA219 boards are bad. I have stopped buying the purple ones as most of them sold don’t work. If they don’t test OK, they won’t work when soldered in. 

I have had better luck with the blue INA219 boards, but they are bigger and require some mods to fit.

Good luck with your build!

Alan

**Comment by gorkaarce** [comment]
hi alan,
 thanks for the answer, 
we have used the blue INAs (the picture is from the wiki to ilustrate the test). 
None of them are testing as ok on the test pins, but work fine, the trouble is with the SW in my opinion.
and on both cubesats we mounted the same trouble related to the same bus 1..(that´s why i posted it there)
we just received the sparkfun pro micro (really hard to get the whole BOM..)and finished testing hopefully everything works, if you have any feedback regarding the bus 1 issue..would be great to know.
cheers
gorka
